sa/_db-next/migrations/20190902000000_AddPrecertificatesTable.sql

```diff
@@ -0,0 +1,33 @@
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+CREATE TABLE `serials` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `registrationID` bigint(20) NOT NULL,
+  `serial` varchar(255) NOT NULL,
+  `created` datetime NOT NULL,
+  `expires` datetime NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `serial` (`serial`),
+  KEY `regId_serials_idx` (`registrationID`),
+  CONSTRAINT `regId_serials` FOREIGN KEY (`registrationID`) REFERENCES `registrations` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION
+) DEFAULT CHARSET=utf8;
```

Comment:
I think this should also have a `ENGINE=InnoDB` to match other table creation migrations.

Reply:
Done :rocket: